devices/Paexo/src/Paexo.cpp

```diff
@@ -359,6 +469,70 @@ class Paexo::PaexoImpl::PaexoTorque3DSensor : public wearable::sensor::ITorque3D
     }
 };
 
+// ===========================================
+// Paexo implementation of ForceTorque6DSensor
+// ===========================================
+class Paexo::PaexoImpl::PaexoForceTorque6DSensor : public wearable::sensor::IForceTorque6DSensor
+{
+public:
+    Paexo::PaexoImpl* paexoImpl = nullptr;
+
+    PaexoForceTorque6DSensor(Paexo::PaexoImpl* impl,
+                             const wearable::sensor::SensorName name = {},
+                             const wearable::sensor::SensorStatus status = wearable::sensor::SensorStatus::Ok) // Default sensor status is Ok, to be updated using node status from iFeelDriver
+        : IForceTorque6DSensor(name, status)
+        , paexoImpl(impl)
+    {
+        // TODO: Initialization
+    }
+
+    ~PaexoForceTorque6DSensor() override = default;
+
+    void setStatus(const wearable::sensor::SensorStatus status) { m_status = status; }
+
+    bool getForceTorque6D(Vector3& force3D, Vector3& torque3D) const override
+    {
+        assert(paexoImpl != nullptr);
+
+        std::lock_guard<std::mutex> lock(paexoImpl->mutex);
+
+        // Get FT Data from front or back FT using iFeelDriver data
+        // TODO: Cleanup ftData buffer handling
+        paexoImpl->ifeelData = paexoImpl->ifeelDriver->getData();
+
+        if (this->getSensorName().find("Left") != std::string::npos)
+        {
+           iFeel::ForceTorque ftData = paexoImpl->ifeelData.FTShoeNodes[2]->getFrontFT();
```

Comment:
it looks like the node number here is hardcoded, you could use the one that you are passing in the device configuration `ifeeldriver-config-nodes`.

Reply:
Done in https://github.com/robotology/wearables/pull/117/commits/91932be4ea6c84176e597b9f4f60135c96c54350

Also,  added a check during `iFeelDriver` configuration https://github.com/robotology/wearables/blob/91932be4ea6c84176e597b9f4f60135c96c54350/devices/Paexo/src/Paexo.cpp#L279-L280